packages/terra-clinical-header/src/Header.scss

```diff
@@ -46,10 +47,10 @@
     margin-top: 0;
     overflow: hidden;
     text-overflow: ellipsis;
-    white-space: nowrap;
+    white-space: normal;
     width: 100%;
-    word-break: normal;
-    word-wrap: normal;
+    word-break: break-word;
+    word-wrap: break-word;
   }
```

Comment:
`it cannot uncollapse the elements from the dropdown.`

This was taken from the pull request description above. Were you able to find a solution for this?

Reply:
Not without breaking other scenarios.  Resizing the window down and back bigger again is something that I feel is a rare enough use case that it is ok.

Reply:
I think a screen rotation could trigger this situation

Reply:
updated to handle the resizing by setting the widths on resize now